product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/index.mdx

```diff
@@ -16,11 +16,12 @@ You can modify your cluster by modifying:
 
 You can also modify your cluster by installing Postgres extensions. See [Postgres extensions](/biganimal/release/using_cluster/03_modifying_your_cluster/extensions) for more information.
 
-!!! Note
-    You can't switch from a single node cluster or a high availability cluster to an extreme high availability cluster or vice versa.
 
 ## Modify your cluster
 
+!!! Note
+    Any changes made to the cluster's instance type, volume type or volume properties are not automatically applied to its replicas's settings. To avoid lag during replication, ensure that your replica is at least as capable as its parent cluster. See, 
```

Comment:
@moiznalwalla , were you intending to add a cross reference here?

Reply:
Fixed.